packages/DataTransformation/models/marts/planlimits/groundwater_limits.sql

```diff
@@ -0,0 +1,61 @@
+{{ config(
+    materialized = 'view'
+) }}
+
+WITH council_plan_documents AS (
+
+  SELECT * FROM {{ ref('stg_planlimits_council_plan_documents') }}
+),
+
+plans AS (
+
+  SELECT * FROM {{ ref('plans') }}
+),
+
+plan_regions AS (
+
+  SELECT * FROM {{ ref('plan_regions') }}
+),
+
+temp_plan_regions AS (
+
+  SELECT
+
+    p.council_id,
+    p.id AS plan_id,
+    JSONB_ARRAY_ELEMENTS(document -> 'regions') AS region
+
+  FROM council_plan_documents AS cpd
+
+  INNER JOIN plans AS p ON cpd.council_id = p.council_id
+),
```

Comment:
Can we define temp-plan_regions as a macro to avoid future errors?  There may be some other macro candidates too?